README.md

```diff
@@ -175,12 +175,8 @@ _This API is influenced by gRPC's [Java library](https://github.com/grpc/grpc-ja
 ### Concurrency limits
 
 Every request passes through a pair of [AIMD](https://en.wikipedia.org/wiki/Additive_increase/multiplicative_decrease)
-concurrency limiters.
-There are two types of concurrency limiter: per-host, and per-endpoint. The former is based on failures that
```

Comment:
moved this:

> The former is based on failures that
indicate the target host is in a degraded state, and the latter is based on failures that are coupled to an individual endpoint.

below to those specific sections.  This way the `### Concurrency limits` section is in steadily increasing level of detail, instead of putting this up in the summary